src/app/project/compliance/inspection-detail/inspection-detail.component.html

```diff
@@ -50,11 +50,56 @@ <h3 for="notes">Notes: </h3>
   </section>
   <section>
     <div *ngIf="elements.length > 0" class="label-pair">
-      <h3 for="tags">Inspection Submissions: </h3>
-      <app-table-template *ngIf="!loading && tableParams.totalListItems != 0" (checkboxChange)='checkChange($event)'
-        [columns]="tableColumns" [data]="tableData" (onColumnSort)='setColumnSort($event)'
-        (onSelectedRow)='updateSelectedRow($event)' (onPageNumUpdate)='getPaginatedDocs($event, null, null)'>
-      </app-table-template>
+      <h3 for="tags">Inspection Submissions</h3>
```

Comment:
"for" is not a valid attribute for `h3` elements. I know you didn't write this, but I have been stripping them out as I see them.

Reply:
Thank you! I will get rid of that one.